### PR TITLE
TF: update down camera tf

### PR DESCRIPTION
**Pull request by DSsoto**
This is the tf that I used to get the path alignment mission to work last pool day. 

Did not actually look at the camera positioning inside the box but all other rotations of 90 degrees that keep the camera facing down do not work.

### CODE REVIEW

File: command/sub8_launch/launch/tf.launch [hunk 1]
```diff
@@ -13,7 +13,7 @@
     </node>
 
     <node pkg="tf" type="static_transform_publisher" name="camera_down_broadcaster"
-      args="0.0537  0.1048 -0.2565   7.07106781e-01 -7.07106781e-01 0.0 0.0 base_link down_left_cam 100">
+      args="0.054 0.105 -0.256 1 0 0 0 0 base_link down_left_cam 100">
```
Comment (kev-the-dev) [inline command/sub8_launch/launch/tf.launch]: * this round up the translation, which might not be accurate
* invalid syntax, there are too many numbers here

**Review by kev-the-dev — CHANGES_REQUESTED** [review]

File: command/sub8_launch/launch/tf.launch (hunk 1 above)
Comment (DSsoto) [reply command/sub8_launch/launch/tf.launch]: Sorry, one zero too many.

Even saying that we know this measurement to the order of millimeters is really pushing it. We should really not use more than four significant figures for any of these measurements

**Review by kev-the-dev — CHANGES_REQUESTED** [review]
Also please add the following patch so the change is reflected in gazebo (I tested it)
```
diff --git a/simulation/sub8_gazebo/models/sub8/sub8.sdf b/simulation/sub8_gazebo/models/sub8/sub8.sdf
index b1c9d33..fd94525 100644
--- a/simulation/sub8_gazebo/models/sub8/sub8.sdf
+++ b/simulation/sub8_gazebo/models/sub8/sub8.sdf
@@ -105,7 +105,7 @@
 
       <sensor name="down_camera" type="camera">
         <camera>
-          <pose>0.0537 0.1048 -0.2565  0 1.57 0</pose>
+          <pose>0.0537 0.1048 -0.2565 -1.57079632679 1.57079632679 0.0 </pose>
           <horizontal_fov>1.047</horizontal_fov>
           <image>
             <width>644</width>
```

**Comment by DSsoto** [comment]
Done.

**Review by kev-the-dev — APPROVED** [review]

**Pull request merged**